src/Commands/ShowModelCommand.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace FumeApp\ModelTyper\Commands;
+
+use Illuminate\Foundation\Console\ShowModelCommand as BaseCommand;
```

Comment:
I am confused, should it be:
```php
use Illuminate\Database\Console\ShowModelCommand as BaseCommand;
```
feel free to update the `lluminate/database` requirement if needed

Reply:
- For Laravel < 10, the correct command is `Illuminate\Foundation\Console\ShowModelCommand`
- For Laravel 10+, the correct command is `Illuminate\Database\Console\ShowModelCommand`

The current version requirement of `"illuminate/database": "^9.45"` of master is Laravel 9 compatible.

Reply:
any reason not to update:
```
"illuminate/database": "^10.00"
```
does that make it easier?

Reply:
i'd rather just support the latest version and not have to conflict with older (even if older was just recently updated)

Reply:
> i'd rather just support the latest version and not have to conflict with older (even if older was just recently updated)

In that case we can change the class to `Illuminate\Database\Console\ShowModelCommand` and db version to `"illuminate/database": "^10.00"` as per your suggestion. The database requirement should also be moved from `require-dev` to `require`.

Reply:
ok sounds good, feel free to make that change

Reply:
besides that everything is looking/working great